benchmarks/benchmark.py

```diff
@@ -0,0 +1,92 @@
+"""Quantifies the performance of different centroid generation techniques
+
+To measure how well a generation technique, ie. random centroids, CVT, etc,
+performs, we measure the probability of generating a random point within a
+certain region defined by the centroid of that region.
+
+The mathematical equations and foundations of this benchmarking technique is
+found within the paper: https://dl.acm.org/doi/pdf/10.1145/3583133.3590726.
+
+Usage:
+    python benchmarks.py
+
+This script will generate centroids using 2 techniques, CVT and random
+generation. These centroids will then be evaluated by the get_score()
+function which will output a probability score between [0, 1].
+"""
+
+import numpy as np
+from scipy.spatial import distance
+
+from ribs.archives import CVTArchive
+
+
+def get_score(centroids, num_samples, seed):
+    """Returns the performance of generated centroids
+
+    Args:
+        centroids (numpy.array): centroids being evaluated
```

Comment:
should be numpy.ndarray (numpy.array is a function that creates an array, and numpy.ndarray is the actual type)